alcs-frontend/src/app/shared/meeting-overview/meeting-overview.component.scss

```diff
@@ -102,12 +102,12 @@ mat-panel-description {
 :host::ng-deep .mat-expansion-indicator::after {
   border-width: 0 4px 4px 0;
   padding: 5px;
-  color: $black;
+  color: colors.$black;
 }
 
 .main-panel-no-meetings {
-  @extend h4;
-  color: $grey;
+  //FIX @extend typography, h4;
```

Comment:
is it still relevant?

Reply:
Yes, fixed now! I thought I wrote it as //TODO and couldn't find it.